.github/workflows/preview-teardown.yml

```diff
@@ -33,7 +33,6 @@ jobs:
         body-includes: Preview is deployed to 
 
     - name: Teardown preview on Surge
-      if: ${{ steps.fc.outputs.comment-id != 0 }}
```

Comment:
I enabling dependabot worked then we can restore this right?

Reply:
No - you enable Dependabot by creating `dependabot.yml`. It is by design that Dependabot only has access to a read-only Github Token. Though we can modify the permissions given by the Github Token and allow limited write access - I will have to test more with it.

Reply:
I thought there was a setting to give dependabot permissions from the settings

Reply:
I think the only ones there are to enable Dependabot's security alerts? Lmk if I missed something.

Reply:
Ok then I think we are good not a big deal, especially since there is a deployment link anyways in the action's log